src/features/cartography/infrastructure/configuration/modules/cartography.feature.module.ts

```diff
@@ -24,6 +36,10 @@ import { GeocodeAddressUseCase } from '../../../use-cases/geocode-address/geocod
     },
     CnfsRest,
     CoordinatesRest,
+    {
+      provide: ClusterService,
+      useFactory: (): ClusterService => new ClusterService()
```

Comment:
Du coup, je pense que tu peux faire :
```suggestion
      useClass: ClusterService
```